Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryISISLoadAndProcess.py

```diff
@@ -190,12 +193,19 @@ def _declareReductionProperties(self):
             'MonitorIntegrationWavelengthMin', 'MonitorIntegrationWavelengthMax',
             'SubtractBackground', 'BackgroundProcessingInstructions', 'BackgroundCalculationMethod',
             'DegreeOfPolynomial', 'CostFunction',
-            'NormalizeByIntegratedMonitors', 'PolarizationAnalysis',
+            'NormalizeByIntegratedMonitors', 'PolarizationAnalysis'
+        ]
+        self._copy_properties_from_reduction_algorithm(properties)
+
+        self.declareProperty('PolarizationEfficiencies', "",
+                             'A workspace or file name containing the polarization efficiency factors for either '
+                             'the Wildes or Fredrikze correction methods.', Direction.Input)
```

Comment:
I might be missing something, but is there a reason we don't just copy this property from `ReflectometryReductionOneAuto`?

Reply:
Unfortunately in this case they're different types - this one needs to take a string while the property on `ReflectometryReductionOneAuto` takes a workspace.

Reply:
Ah, I see. That makes sense 👍